src/routing/tests/mod.rs

```diff
@@ -569,7 +571,7 @@ async fn handle_unknown_message(source: UnknownMessageSource) -> Result<()> {
     let original_message = Message::single_src(
         &sender_keypair,
         MIN_AGE + 1,
-        DstLocation::Section(rng::new().gen()),
+        DstLocation::Section(SmallRng::from_entropy().gen()),
```

Comment:
just wondering, why sometime OsRng, sometime SmallRng ?

Reply:
One is a secure rng and the other is not. The secure rng is much more resource-hungry and best to only use it for csprng

Reply:
thx for the clarification